lib/connect-dynamodb.js

```diff
@@ -106,7 +106,7 @@ module.exports = function (connect) {
     DynamoDBStore.prototype.get = function (sid, fn) {
 
         sid = this.prefix + sid;
-        var now = +new Date;
+        var now = Math.floor(Date.now() / 1000);
```

Comment:
You are right, I haven't tested against tables with rows using milliseconds. Probably some weird stuff would happen if sec/millis were mixed :)

I guess the safest way is to define a maintenance window to drop the table and let the connector recreate it and start using expires as seconds.